pkg/tests/support/helpers.py

```diff
@@ -621,18 +363,22 @@ def _install_pkgs(self, upgrade=False):
             ret = self.proc.run(self.pkg_mngr, "install", "-y", *self.pkgs)
         if not platform.is_darwin() and not platform.is_windows():
             # Make sure we don't have any trailing references to old package file locations
-            ret.returncode == 0
+            assert ret.returncode == 0
             assert "/saltstack/salt/run" not in ret.stdout
         log.info(ret)
         self._check_retcode(ret)
 
-    def install(self, upgrade=False):
-        if self.compressed:
-            self._install_compressed(upgrade=upgrade)
-        else:
-            self._install_pkgs(upgrade=upgrade)
-            if self.distro_id in ("ubuntu", "debian"):
-                self.stop_services()
+    def package_python_version(self):
+        return self.proc.run(
+            str(self.binary_paths["python"][0]),
+            "-c",
+            "import sys; print('{}.{}'.format(*sys.version_info))",
+        ).stdout.strip()
+
+    def install(self, upgrade=False, downgrade=False):
+        self._install_pkgs(upgrade=upgrade, downgrade=downgrade)
+        if self.distro_id in ("ubuntu", "debian"):
+            self.stop_services()
```

Comment:
```suggestion
        if self.distro_id in ("ubuntu", "debian"):
            self.stop_services()
```
Any particular reason why we're avoiding the OS default behavior of starting services(or leaving them running)?

Reply:
Closing, discussed in person